salt/states/netacl.py

```diff
@@ -428,7 +427,11 @@ def term(name,
             - filter_options:
                 - not-interface-specific
             - term_name: {{ term_name }}
-            - {{ my_term_cfg }}
+            - {{ my_term_cfg | json }}
+
+    When passing retrieved pillar data into the state file, it is strongly
+    recommended to use the json serializer explicitly, instead of relying
```

Comment:
No, I meant some enhancements to the text you already have:

- ```When passing retrieved pillar data into the state file, it is strongly
    recommended to use the json serializer explicitly (``| json``), instead of relying
    on the default Python serializer.```

OR

- ```When passing retrieved pillar data into the state file, it is strongly
    recommended to use the json serializer explicitly (see examples below), instead of relying
    on the default Python serializer.```

I let you chose what's the most obvious.